### DISCUSSION TITLE
How to add class weights for object detection?

### DISCUSSION THREAD

**Discussion by gH2AX** [keras-team/keras-cv#1753] [Q&A]
Dear keras-cv Team,

I had my first little sense of achievement with your RetinaNet model and now want to improve upon that.
My images contains an unbalanced number of objects from two classes.
When I applied semantic segmentation to the same problem, I had great success with feeding modified class weights to the training process. How can this be done when using an object detection model from your library?

Many thanks in advance!

Edit: I just realized that's being addressed by the focal loss function.
